appcelerator/titanium_mobile
# [TIMOB-24008] Added validation to prevent managed provisioning profiles from being used.

**JIRA:** https://jira.appcelerator.org/browse/TIMOB-24008

To test:
1. Make sure you have at least one managed provisioning profile installed where the provisioning profile must contain "iOS Team Provisioning Profile".
2. Run `ti info -t ios` and verify that it prints that it is unsupported. 
3. Run `ti build -p ios -T device -P foo` and make sure that managed pp-uuids are not listed in prompt.
4. Run `ti build -p ios -T device -P <managed-pp-uuid>` and make sure the build errors with `[ERROR] Specified provisioning profile UUID "<managed-pp-uuid>" is managed and not supported`.
